README.md

```diff
@@ -31,7 +31,7 @@
 ## Installation
 
 ```bash
-go get github.com/imgix/imgix-go/v2
+go get github.com/imgix/imgix-go/v2@v2.0.1
```

Comment:
Is there a way to specify this so you don't have to update it for every release?
Does `go get` support something like `github.com/imgix/imgix-go@main`?

Reply:
Looks like the answer here is "no"